R/conda.R

```diff
@@ -373,6 +375,14 @@ conda_version <- function(conda = "auto") {
   system2(conda_bin, "--version", stdout = TRUE)
 }
 
+
+numeric_conda_version <- function(conda = "auto") {
+  v <- conda_version(conda)
+  v <- sub("(conda) ([0-9.]+)", "\\2", v)
```

Comment:
Do the parentheses in `(conda)` need to be escaped? Or are you intentionally creating (and discarding) this capture?

Reply:
The latter